src/mower_logic/src/monitoring/monitoring.cpp

```diff
@@ -67,6 +67,8 @@ ros::NodeHandle *n;
 ros::Time last_status_update(0);
 ros::Time last_pose_update(0);
 
+int currentMowingAreaPublic;
```

Comment:
not used?

Reply:
yes, that was earlier version, and I removed it in favour of doing a get_current_area() method instead which is the way I'm now trying to implement it.